parser/src/command/assign.rs

```diff
@@ -14,9 +14,13 @@ use std::fmt;
 
 #[derive(PartialEq, Eq, Debug)]
 pub enum AssignCommand {
+    /// Corresponds to `@bot claim`.
     Own,
+    /// Corresponds to `@bot release-assignment`.
     Release,
+    /// Corresponds to `@bot assign @user`.
     User { username: String },
+    /// Corresponds to `r? [@]user`.
     ReviewName { name: String },
```

Comment:
:heart: 

We should also think about renaming them to their actual command name. `Own` is particularly inscrutable without the command.

Reply:
Done that!